installGroot/install.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="/installGroot.html/install.css" rel="stylesheet">
+    <title>Install Groot</title>
+</head>
+<body>
+    <div class="groot">
+        <div class="groot-container first">
+            <nav>
+            <img src="/images/groot.png" style="width: 4rem;height: auto; margin: 0 0.5rem "/>
+            <span style="display: inline-flex;"><a href="/" class="square">GrooT UI</a></span>
+            <div class="version"> v1.0</div>
+            </nav>
+            <div class="item-container">
+                <div class="install-ui"><a href="/installGroot.html/install.html" class="install-text"> Installation </a></div>
+                        <a href="/Components/avatar/avatar.html" class="avatar-text ">Avatars</a>
+                        <a href="/Components/alert/alert.html" class="avatar-text">Alert</a>
+                        <a href="/Components/badges/badges.html" class="avatar-text">Badges</a>
+                        <a href="/Components/buttons/button.html" class="avatar-text">Buttons</a>
+                        <a href="/Components/card/card.html" class="avatar-text">Cards</a>
+                        <a href="/Components/modal/modal.html" class="avatar-text">Modals</a>
+                        <a href="/Components/input/input.html" class="avatar-text">Input</a>
```

Comment:
I would suggest aligning the component names to the left for them to look more even.

Reply:
Have done the necessary changes mentioned by. Thank you for your valuable suggestion.